package/csac_daemon/src/serial_utils.h

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
This feels like common code that could be moved to libpiksi?

Reply:
I definitely agree with this. let's put this in `libpiksi` and refactor `cell_modem_daemon` to use these (if possible)